CHANGELOG.md

```diff
@@ -1,3 +1,40 @@
+## 15.5.2
+
+_released `2024-02-05`_
+
+### cli
+
+- :wrench: Updated the "custom-auth" sample to avoid a warning when running zapier validate ([#724](https://github.com/zapier/zapier-platform/pull/724))
+
+- :bug: Fixed bug in `zapier convert` that crashes due to syntax error in user's code and should not replace `source` in sample ([#730](https://github.com/zapier/zapier-platform/pull/730))
+
+- :wrench: Improved handling of broken symlinks while copying files to temp directory during the build process ([#737](https://github.com/zapier/zapier-platform/pull/737))
+
+- :bug: `zapier build` fails with npm workspaces ([#738](https://github.com/zapier/zapier-platform/pull/738))
+
+- :wrench: Added @clif/core dependency, upgraded @oclif/plugin-help ([#739](https://github.com/zapier/zapier-platform/pull/739))
```

Comment:
This reminds me that since we no longer import `@oclif/core`, can we remove the dependency from `package.json`?